### PR TITLE
Restyle landing section to be consistent with Figma

### PR DESCRIPTION
**Pull request by tmacsmee**
Didn't add buttons to 'about' section cards, would need a `url` property from sanity for this.

### CODE REVIEW

**Comment by tmacsmee** [comment]
Looks like the background video is taking a while to load on preview. I'll add a placeholder before merging.

**Review by bcho892 — COMMENTED** [review]
styling looks good, however just some findings...

File: client/public/videos/hero-background.mp4
[the remark this answers is not in the document]
Comment (bcho892) [reply client/public/videos/hero-background.mp4:1]: 40MB seems quite bad even for a video

[the remark this answers is not in the document]
Comment (tmacsmee) [reply client/public/videos/hero-background.mp4:1]: Yeah there's probs a better way, might try to convert to webm. You might have to handle CDN stuff cos I'm not sure what to do there.

**Review by bcho892 — APPROVED** [review]
ok seems like the video at least caches now so gj!

**Pull request merged**